spec/system/delete_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
+  let!(:kudo) { create(:kudo, giver: employee) }
+  let(:employee) { create(:employee) }
```

Comment:
order of these two lines is wrong
first, you should create `employee` that later you substitute for `kudo`